### PR TITLE
Configure new languages [waiting for translations]

### PR DESCRIPTION
**Pull request by danlaush**
## Context

We need some new languages for [rate alerts](https://transferwise.com/tools/exchange-rate-alerts/), which should be accessible for any users of [currency converter](https://transferwise.com/gb/currency-converter/) where you can sign up for a rate alert. CC has quite a wide language set ([crowdin](https://crowdin.com/project/transferwise-currency-converte)).

[Track translations progress on crowdin](https://crowdin.com/project/public-navigation)

## Changes

Update config to match languages set up in crowdin. Adds bg, ms, nn, sk and removes unused es_MX, et

## Checklist
- [x] I have considered the impact of this change and added a [Change Classification](
https://transferwise.atlassian.net/wiki/spaces/EKB/pages/1401189673/Change+Classifications+and+Expectations) Label (`change:standard`, `change:impactful`, `change:emergency`)
- [x] Change meets or does not compromise the [Baseline Security Requirements](https://transferwise.atlassian.net/wiki/spaces/EKB/pages/434929973/Baseline+Security+Requirements)

### CODE REVIEW

File: crowdin.yml [hunk 1]
```diff
@@ -10,26 +10,28 @@ files: [
...
   "languages_mapping" : {
     "locale" : {
+      "bg": "bg",
       "cs": "cs",
       "de": "de",
       "el": "el",
       "en-GB": "en",
       "en-US": "en_US",
       "es-ES": "es",
-      "es-MX": "es_MX",
```
Comment (KarlEST) [inline crowdin.yml:19]: We still have `es_MX` and `et` inside `Homepage` but we don't support them from language select. 🤔  Wondering if we should remove them from there as well or why do we even have them in the first place if we don't use them. Could it be that we want to release these languages in the future and we are already translating them (they have been there for 2 years so I doubt it)?

Comment (danlaush) [reply crowdin.yml:19]: Yeah I assume that's where the list came from originally. The public-navigation project in crowdin doesn't have them as supported languages so there have never been translations for them. What does the homepage crowdin look like? Is that what you mean, that es_MX and et are translated in the homepage project?

Comment (KarlEST) [reply crowdin.yml:19]: Oke, seems like in Crowdin they are not translated but they are included in our `crowdin.yaml`. I'll remove them from there as well. 👍

**Review by KarlEST — APPROVED** [review]

**Comment by danlaush** [comment]
I'm going to leave this PR open until the translations are complete, so it's clear these languages aren't actually supported yet.

**Comment by danlaush** [comment]
Will re-do this. PR is quite out of date.

**Pull request closed without merging**